### ISSUE TITLE
DeployNewAppError:ERROR: (gcloud.app.deploy) An error occurred while parsing file: [/usr/local/google/home/epad/django-project-test/app.yaml]Unable to assign value 'None' to attribute 'service':Value 'None' for service does not match expression '^(?:^(?!-)[a-z\d\-]{0,62}[a-z\d]$)$'  in "/usr/local/google/home/epad/django-project-test/app.yaml", line 2, column 10 during "django-cloud-deploy new"

### ISSUE THREAD

**Issue by enriquejosepadilla** [GoogleCloudPlatform/django-cloud-deploy#282]
## Issue running command
django-cloud-deploy new

## Environment information

Django Cloud Deploy: [0.1.6]

Platform: [Linux-4.19.20-1rodete1-amd64-x86_64-with-Debian-rodete-rodete]

Gcloud Version: [231.0.0]

Cloud SQL Proxy Version: [Cloud SQL Proxy: version 1.13; sha ac834ce67862b7c4ed8bdbe86e664572f820c245 built Tue Oct  9 23:39:16 UTC 2018]

Python Version: [3.5.4rc1 (default, Jul 25 2017, 08:53:34)  [GCC 7.3.0]]

Docker Version: [Docker version 18.06.2-ce, build 6d37f41]

## Traceback
```.pytb
Traceback (most recent call last):
  File "/usr/local/google/home/epad/Projects/prompt/django-cloud-deploy/django_cloud_deploy/django_cloud_deploy.py", line 36, in _new
    new.main(args)
  File "/usr/local/google/home/epad/Projects/prompt/django-cloud-deploy/django_cloud_deploy/cli/new.py", line 157, in main
    backend=args.backend)
  File "/usr/local/google/home/epad/Projects/prompt/django-cloud-deploy/django_cloud_deploy/workflow/__init__.py", line 244, in create_and_deploy_new_project
    project_id, django_directory_path, is_new=is_new)
  File "/usr/local/google/home/epad/Projects/prompt/django-cloud-deploy/django_cloud_deploy/workflow/deploy_workflow.py", line 50, in deploy_gae_app
    region, is_new)
  File "/usr/local/google/home/epad/Projects/prompt/django-cloud-deploy/django_cloud_deploy/workflow/_deploygae.py", line 101, in deploy_gae_app
    raise DeployNewAppError(gcloud_result.stderr)
django_cloud_deploy.workflow._deploygae.DeployNewAppError: ERROR: (gcloud.app.deploy) An error occurred while parsing file: [/usr/local/google/home/epad/django-project-test/app.yaml]
Unable to assign value 'None' to attribute 'service':
Value 'None' for service does not match expression '^(?:^(?!-)[a-z\d\-]{0,62}[a-z\d]$)$'
  in "/usr/local/google/home/epad/django-project-test/app.yaml", line 2, column 10


```

**Comment by enriquejosepadilla** [comment]
@wangxf123456 My app.yaml has service: None, which seems to throw an issue.

**Comment by wangxf123456** [comment]
This should be fixed by #279
